Zend/zend_object_handlers.c

```diff
@@ -2024,14 +2025,29 @@ ZEND_API ZEND_COLD bool zend_std_unset_static_property(zend_class_entry *ce, zen
 
 static ZEND_COLD zend_never_inline void zend_bad_constructor_call(zend_function *constructor, zend_class_entry *scope) /* {{{ */
 {
+	zend_string *message_suffix = ZSTR_EMPTY_ALLOC();
+
+	if (zend_attribute_get_nonpublic_suffix(constructor->common.attributes, &message_suffix) == FAILURE) {
+		return;
```

Comment:
could you please add the same assertion here that you added everywhere else when the above operation fails?